doc/modules/lda_qda.rst

```diff
@@ -8,13 +8,12 @@ Linear and quadratic discriminant analysis
 
 Linear discriminant analysis (:class:`lda.LDA`) and
 quadratic discriminant analysis (:class:`qda.QDA`)
-are two classic classifiers, with, as their names suggest, a linear and a
```

Comment:
Ok, the only thing is that "classic classifiers" clashes clunkily... What about "standard classifiers" ?